configs/hoefling_2024_core_readout_high_res.yaml

```
@@ -34,6 +34,7 @@ model:
   # downsample_input_kernel_size: [1, 4, 4]  # to reproduce 18x16px lowres input
 
 paths:
+  # load_model_path: # e.g. hoefling_2024_base_high_res or path to a .ckpt model
```

```suggestion
  load_model_path: None  # e.g. hoefling_2024_base_high_res or path to a .ckpt model
```

I'd explicitly set this to None, as then it's not commented out. Also in the config below.